include/zim/meson.build

```diff
@@ -12,8 +12,7 @@ install_headers(
     'suggestion.h',
     'suggestion_iterator.h',
     'version.h',
-    zim_config,
-    subdir:'zim'
```

Comment:
This change should be reversed. See the meaning of the `subdir` option of the `install_headers` operation: https://mesonbuild.com/Reference-manual_functions.html#install_headers

Reply:
I will revert this commit